roles/onboarding_controllers/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: Add vSmart devices
   cisco.catalystwan.devices_controllers:
-    device_ip: "{{ device_item.transport_public_ip }}"
```

Comment:
Do you know for which versions it is true?

Because for some time we have been using transport public ip to add controllers, and it worked, so it would be beneficial to check sdwan versions for these cases - that just looks suspicious :)

Reply:
Strangely it looks like both versions work. I did some re-tests and decided to switch it back as it was. I'll need to re-check Azure though when done with the NSG policy related issue.